docs/samples/specification/azure_key_credential/generated/azure/key/credential/sample/__init__.py

```diff
@@ -10,9 +10,16 @@
 from ._version import VERSION
 
 __version__ = VERSION
+
+try:
+    from ._patch import __all__ as _patch_all
+    from ._patch import *  # pylint: disable=unused-wildcard-import
+except ImportError:
+    _patch_all = []
+from ._patch import patch_sdk as _patch_sdk
 __all__ = ['AutoRestHeadTestService']
+__all__.extend(_patch_all)
```

Comment:
if  `_patch_all` also has `AutoRestHeadTestService`, __all__ will be like ["AutoRestHeadTestService", "AutoRestHeadTestService"]. It is a little confusing although actually only the latter works which is expected.

Reply:
that's a really good point, i'm going to add some tests and fix this, then report back. thanks for the great catch!

Reply:
added [this test](https://github.com/Azure/autorest.python/blob/inherit_customization/test/dpg/version-tolerant/AcceptanceTests/test_customizations.py#L79) in sync and async, thanks for the catch!